### PR TITLE
WLA-605: SYA e2e Phani

### PR DESCRIPTION
**Pull request by phaniperla**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

https://tools.hmcts.net/jira/browse/WLA-605

E2E SYA test

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: test/e2e/page-objects/cya/checkYourAppeal.js [hunk 1]
```diff
@@ -21,8 +21,14 @@ const appellant = testDataEn.appellant;
 function enterDetailsFromStartToNINO(commonContent, language, benefitTypeCode = testDataEn.benefitType.code) {
   const I = this;
 
-  I.enterBenefitTypeAndContinue(commonContent, benefitTypeCode);
-  // I.chooseLanguagePreference(commonContent, 'no');
+  if (language === 'cy') {
+    I.click({ xpath: '/html/body/div[2]/div/p/span/a[2]' });
```
Comment (LucaDelBuonoHMCTS) [inline test/e2e/page-objects/cya/checkYourAppeal.js:25]: Looks like this is failing in Jenkins.

Comment (phaniperla) [reply test/e2e/page-objects/cya/checkYourAppeal.js:25]: Welsh hyper link is disabled on AAT/PR , updated code, now this time it should work

Comment (LucaDelBuonoHMCTS) [reply test/e2e/page-objects/cya/checkYourAppeal.js:25]: I wouldn't access it via the full DOM path, better if you add an ID to the element and access it with that.

Comment (phaniperla) [reply test/e2e/page-objects/cya/checkYourAppeal.js]: taken care reviewer comments

**Pull request closed without merging**